lib/modules/platform/gerrit/types.ts

```diff
@@ -0,0 +1,163 @@
+  permitted_labels?: any[];
+  removable_reviewers?: any[];
+  reviewers?: Record<GerritReviewersType, GerritAccountInfo[]>; //key = ReviewerState
+  pending_reviewers?: any;
+  reviewer_updates?: any;
+  messages?: GerritChangeMessageInfo[];
+  current_revision?: string;
+  revisions?: Record<string, GerritRevisionInfo>; //All patch sets of this change as a map that maps the commit ID of the patch set to a RevisionInfo entity.
+  meta_rev_id?: string;
+  tracking_ids?: any;
+  _more_changes?: any;
+  problems: any[];
+  is_private?: boolean;
+  work_in_progress?: boolean;
+  has_review_started?: boolean;
+  revert_of?: string;
+  submission_id?: string;
+  cherry_pick_of_change?: string;
+  cherry_pick_of_patch_set?: string;
+  contains_git_conflicts?: string;
+}
+
+//type ReviewerState = "REVIEWER"|"CC"|"REMOVED";
+export interface GerritRevisionInfo {
+  kind: string; //The change kind. Valid values are REWORK, TRIVIAL_REBASE, MERGE_FIRST_PARENT_UPDATE, NO_CODE_CHANGE, and NO_CHANGE.
+  _number: string; //The patch set number, or edit if the patch set is an edit.
+  created: Date; //TODO: map
+  uploader: GerritAccountInfo;
+  ref: string; //The Git reference for the patch set.
+  actions?: Record<string, GerritActionInfo>;
+  //... many more...still not necessary
+}
+
+export interface GerritChangeMessageInfo {
+  id: string;
+  author?: GerritAccountInfo;
+  real_author?: GerritAccountInfo;
+  date: Date; //TODO: convert
+  message: string;
+  accountsInMessage: GerritAccountInfo[];
+  tag?: string;
+  _revision_number?: number;
+}
+
+export interface GerritLabelInfo {
+  optional?: boolean;
+  description?: string;
+  approved?: GerritAccountInfo;
+  rejected?: GerritAccountInfo;
+  recommended?: GerritAccountInfo;
+  disliked?: GerritAccountInfo;
+  blocking?: boolean;
+  value?: string; //The voting value of the user who recommended/disliked this label on the change if it is not “+1”/“-1”.
+  default_value?: string;
+  votes?: number[]; //A list of integers containing the vote values applied to this label at the latest patchset.
+  all?: any[];
+  values?: any; //A map of all values that are allowed for this label. The map maps the values (“-2”, “-1”, " `0`", “+1”, “+2”) to the value descriptions.
+}
+
+export interface GerritActionInfo {
+  method?: string;
+  label?: string;
+  title?: string;
+  enabled?: boolean;
+}
+
+export interface GerritAccountInfo {
+  _account_id: number;
+  name?: string;
+  display_name?: string;
+  email?: string;
+  secondary_emails?: string[];
+  username?: string;
+  avatars?: any;
+  _more_accounts?: boolean;
+  status?: string;
+  inactive?: boolean;
+  tags?: string[];
+}
+
+export interface GerritAttentionSetInfo {
+  account: GerritAccountInfo;
+  last_update: Date;
+  reason: string;
+  reason_account: GerritAccountInfo;
+}
+
+export interface GerritMergeableInfo {
+  submit_type: string; //MERGE_IF_NECESSARY, FAST_FORWARD_ONLY, REBASE_IF_NECESSARY, REBASE_ALWAYS, MERGE_ALWAYS or CHERRY_PICK
```

Comment:
you can use string constants instead of generic string here. so type values are limited